cmd/oauth2_implicit.go

```diff
@@ -25,10 +25,13 @@ func (c *OAuth2Cmd) ImplicitGrantFlow(clientConfig oauth2.ClientConfig, serverCo
 
 	LogRequest(authorizeRequest)
 
-	Logfln("\nOpen the following URL:\n\n%s\n", authorizeRequest.URL.String())
+	Logfln("\nGo the following URL:\n\n%s\n", authorizeRequest.URL.String())
```

Comment:
Go to the following URL?

Reply:
A native speaker could probably confirm, but "go to the following URL" is probably more idiomatic (1.1M search results on Google) than "open the following URL" (400k results).
Plus it avoids repeating the word "open" as used later in "opening browser...".